entrypoints/components/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+const HOST_AUTH = 'http://localhost:3000'
+//  import.meta.env.VITE_AUTH_HOST_URL
+
+interface FooterProps {
+  isAuthAvailable: boolean
+  isAuthenticated: boolean
+  onLogOut: () => void
+}
+
+const AuthButtons = ({
+  isAuthenticated,
+  onLogOut,
+}: Pick<FooterProps, 'isAuthenticated' | 'onLogOut'>) => (
+  <div className="flex flex-row w-full items-center gap-2">
+    {isAuthenticated ? (
+      <button
+        className="flex hover:underline bg-transparent text-sm font-medium text-primary"
+        onClick={onLogOut}
+      >
+        Log out
```

Comment:
There may be no plan to add translations, but it is good practice to have the texts in a separate file, for maintainability